### ISSUE TITLE
Dark mode button disappears when dark mode is activated

### ISSUE THREAD

**Issue by HyperTHD** [Seneca-CDOT/telescope#1652]
**What happened**:
When we landed the dark mode switch, it gave Telescope users the ability to run in dark mode. Unfortunately, when pressing the toggle to switch to dark mode, the button looks like it disappeared. It's still there and functionally works to switch back. Just not visible

**What should have happened**:
Toggle to switch back to light theme should be visible

**How to reproduce it (as precise as possible)**:
Simply run the next front-end locally, and press the toggle to switch the theme to dark mode. You'll see the button disappear style wise, but still there and functionally works.

![Toggle visible](https://user-images.githubusercontent.com/16841702/106951451-c9015980-66fd-11eb-8488-db942d0964c2.png)
![Toggle not visible](https://user-images.githubusercontent.com/16841702/106951494-d585b200-66fd-11eb-80ae-6820cc61164e.png)

**Anything else we need to know?**:

**Environment**:

- OS: Windows
- Browser: Google Chrome

**Comment by chrispinkney** [comment]
Is anyone else having this issue? Looks like it's fine on my end using Firefox (which is funny because I was in Abdul's situation the other day haha)


![2021-02-04_15-27-45](https://user-images.githubusercontent.com/7242003/106954456-ce60a300-6701-11eb-8e2b-2a8303933c01.gif)

**Comment by birtony** [comment]
@chrispinkney @HyperTHD I am seeing the same issue on Firefox 85.0 (64-bit) running on Ubuntu 20.04.2 LTS on Netlify - https://telescope-next.netlify.app/
![image](https://user-images.githubusercontent.com/33902374/106958502-72008200-6707-11eb-8ce5-1204ef2393b4.png)

**Comment by humphd** [comment]
I can't reproduce on Firefox/Chrome on macOS.

**Comment by humphd** [comment]
Dug into this a bit with @PedroFonsecaDEV and @birtony and here are our findings:

The button is [here](https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/next/src/components/ThemeToggleButton.tsx):

```tsx
const ThemeToggleButton = ({ theme, toggleTheme }: Props) => {
  const classes = useStyles();
  return (
    <IconButton onClick={toggleTheme} className={classes.themeToggleButton}>
      {theme.palette.type === 'light' ? (
        <Brightness4Icon color="primary" fontSize="large" />
      ) : (
        <Brightness7Icon fontSize="large" />
      )}
    </IconButton>
  );
};
```

Depending on the `theme` we render different icons, and in the first case we set the color to `"primary"` but not in the second.  It's using pure white, so it is invisible if your monitor is wide enough that the button overflows the timeline.

If you look at how `<IconButton>` works in MUI, https://material-ui.com/components/buttons/#icon-buttons, they seem to always provide a `color` of `primary` or `secondary` to pick up from the theme.

I tried adding `color="primary"` to the second one, and it picked up the value from the dark theme, but the colour isn't ideal.

I suspect that we need to do more work on our theme colours and usage.

cc @tonyvugithub, who might have ideas.
